src/main/java/touro/simondrums/SimonGame.java

```diff
@@ -48,6 +51,10 @@ public int getHighScore() {
         return highScore;
     }
 
+    public int getCurrentScore() {
+        return currentScore;
+    }
+
     public void buildSequence() {
```

Comment:
change to `addToSequence()` or something like that which better describes what the method does